src/utilities/lib/monaco.languages.typescript.d.ts

```diff
@@ -0,0 +1,473 @@
+/* eslint-disable */
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we move this files to `src/utilities/types`?

Reply:
Move `monaco.languages.typescript.d.ts` to `src/types`, unless there is a good reason it needs to be under `lib`?

Reply:
So these types are totally fake - they cannot be imported with a standard import because there's just no real JS code at that import path. I'll try to see if they work in src/types though, I thinkkkk that should work? They were in `lib` because it's a magic path that ensures that the modules defined are importable by just their names, but that's it and an import out of types should be fine too I think